File: MyCourses - Android/app/src/main/java/com/mycoursesapp/model/Category.java
package com.mycoursesapp.model;

/**
 * Created by mamdouhelnakeeb on 2/8/18.
 */

public class Category {

    public int id = 0;
    public String name = "";
    public boolean selected = false;
    public String img = "";

    public Category(int id, String name){
        this.id = id;
        this.name = name;
    }

    public Category(int id, String name, String img){
        this.id = id;
        this.name = name;
        this.img = img;
    }

    public Category(){

    }
}

File: MyCourses - iOS/My Courses/Models/Category.swift
//
//  Category.swift
//  My Courses
//
//  Created by Mamdouh El Nakeeb on 3/10/18.
//  Copyright © 2018 My Courses. All rights reserved.
//

import Foundation

struct Category: Codable {
    
    var id = 0
    var name = ""
    var img = ""
    var selected = false
    
    init(id: Int, name: String) {
        
        self.id = id
        self.name = name
    }
    
    init(id: Int, name: String, img: String) {
        
        self.id = id
        self.name = name
        self.img = img
    }
}
